includes/checkout/functions.php

```diff
@@ -27,15 +27,18 @@ function edd_is_checkout() {
 	$is_checkout      = is_page( edd_get_option( 'purchase_page' ) );
 
 	if( ! $is_object_set ) {
-
 		unset( $wp_query->queried_object );
-
+	} else if ( is_singular( apply_filters( 'edd_page_metabox_post_types' , array( 'page' ) ) ) ) {
```

Comment:
This filter name doesn't make sense to me. Is this meant to indicate this is a post type that is supported as an EDD checkout screen?

Reply:
Appears that my previous solution has a left over filter. The goal was to only allow specific types of custom posts to trigger the condition.

Yes. The goal is to make it so that a specific set of post types could trigger this condition.

That filter was left over from having the checkbox in a custom meta box to define something as a checkout page. But since we swayed away from that...thoughts on a better filter name?

Reply:
My initial reaction is that it seems fine to say that a checkout "page" can only be a WP Page.

However, if you want to use a filter to allow other post types to function as a checkout "page", perhaps `edd_checkout_post_types` would be a good filter name.

Reply:
`edd_checkout_post_types` is definitely better.

Reply:
Does it matter what the post type is? Why not just support anything people want to add the shortcode to? The function already has a filter anyway.

Reply:
Yeah I like that.